whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -1447,10 +1453,10 @@ public void lookup_autnum_abuse_contact_from_org() {
                 "[[version, {}, text, 4.0], " +
                 "[fn, {}, text, Abuse Contact], " +
                 "[kind, {}, text, group], " +
-                "[adr, {label=Singel 358}, text, [, , , , , , ]], " +
                 "[tel, {type=voice}, text, +31 6 12345678], " +
                 "[email, {type=email}, text, work@test.com], " +
                 "[email, {type=email}, text, personal@test.com], " +
+                "[adr, {label=Singel 358}, text, [, , , , , , ]], " +
                 "[email, {type=abuse}, text, abuse@test.net]]"));
```

Comment:
Have we changed the order of email and addr? I know its not part of this MR but can you remove the real data from test as well ? Feel free to ignore if it turns out to be lot of chnages

Reply:
I will put the same order as before